servers/visual/rasterizer_rd/rasterizer_scene_high_end_rd.cpp

```diff
@@ -855,7 +852,7 @@ void RasterizerSceneHighEndRD::_render_list(RenderingDevice::DrawListID p_draw_l
 
 		pipeline = &shader->pipelines[cull_variant][primitive][shader_version];
 
-		RD::VertexFormatID vertex_format;
+		RD::VertexFormatID vertex_format = 0;
```

Comment:
That one is a valid bug, this variable and the next two can end up uninitialized if `mesh_surface_get_arrays_and_format` errors out.